minfraud/models.py

```diff
@@ -1075,6 +1075,143 @@ class Subscores:
     }
 
 
+@_inflate_to_namedtuple
+class Reason:
+    """The risk score reason for the multiplier.
+
+    This class provides both a machine-readable code and a human-readable
+    explanation of the reason for the risk score.
+
+    Although more codes may be added in the future, the current codes are:
```

Comment:
Maybe a link to the codes in the dev docs would be useful? I'll be adding it to the node library